addons/mail/static/src/new/core/message_service.js

```diff
@@ -266,7 +272,7 @@ export class MessageService {
         this._updateReactions(message, data.messageReactionGroups);
         this.store.messages[message.id] = message;
```

Comment:
strange that we reassign the store value in update `this.store.messages[message.id] = message;`